src/components/ContentSidebar/DetailsSidebar.js

```diff
@@ -73,13 +71,22 @@ class DetailsSidebar extends React.PureComponent<Props, State> {
     constructor(props: Props) {
         super(props);
         this.state = {
-            file: props.file,
             isLoadingAccessStats: false,
             isLoadingClassification: false,
         };
     }
 
     componentDidMount() {
+        this.fetchFile();
+        this.fetchData();
```

Comment:
The naming here is pretty vague. Can we move the content of `fetchData` directly inside `componentDidMount`, instead?

Reply:
I actually think this might be good to have separated for now, since we will have to implement a `componentDidUpdate`